src/migrations/1747998541677_add-community-members.ts

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { MigrationBuilder, ColumnDefinitions, PgType } from 'node-pg-migrate'
+
+export const shorthands: ColumnDefinitions | undefined = undefined
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  pgm.createTable('community_members', {
+    id: {
+      type: PgType.UUID,
+      primaryKey: true,
+      notNull: true
+    },
+    community_id: {
+      type: PgType.UUID,
+      notNull: true,
+      references: 'communities',
+      onDelete: 'CASCADE'
+    },
+    member_address: {
+      type: PgType.VARCHAR,
+      notNull: true
+    },
+    role_id: {
+      type: PgType.INT,
+      notNull: true,
+      references: 'community_roles',
+      onDelete: 'RESTRICT'
+    },
+    joined_at: {
+      type: PgType.TIMESTAMP,
+      notNull: true,
+      default: pgm.func('now()')
+    },
+    kicked_by: {
+      type: PgType.VARCHAR,
+      notNull: false
+    },
+    kicked_at: {
+      type: PgType.TIMESTAMP,
+      notNull: false
+    },
+    kick_reason: {
+      type: PgType.TEXT,
+      notNull: false
+    }
```

Comment:
This table seems to hold the information about a member of a community. If a community member is kicked, we might want to remove this entry from the table as it does not belong to the community anymore. If we choose not, to see if a user belongs to a community we would need to check for the kicked property, and whenever the user re-joins, we would need to update it properly.
If we want to keep the historic information about a user being kicked, what do you think about having a new table with that information?